test/acceptance/features/bindAppToService.feature

```diff
@@ -330,10 +330,8 @@ Feature: Bind an application to a service
             """
         Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-empty-app" should be changed to "True"
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-empty-app" should be changed to "False"
-        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding "binding-request-empty-app" should be changed to "EmptyApplication"
-        And jq ".status.conditions[] | select(.type=="Ready").status" of Service Binding "binding-request-empty-app" should be changed to "True"
-        And Secret contains "BACKEND_HOST" key with value "example.common"
-        And Secret contains "BACKEND_USERNAME" key with value "foo"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding "binding-request-empty-app" should be changed to "ApplicationNotFound"
```

Comment:
if application selector is not specified, we should not come this far at all - at service binding creation we should get the error.

Reply:
Fixed.